cinema/tests/test_movie_api.py

```diff
@@ -157,3 +159,203 @@ def test_image_url_is_shown_on_movie_session_detail(self):
         res = self.client.get(MOVIE_SESSION_URL)
 
         self.assertIn("movie_image", res.data[0].keys())
+
+
+class UnauthenticatedMovieAPITests(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+
+    def test_auth_required(self):
+        res = self.client.get(MOVIE_URL)
+        self.assertEqual(res.status_code, status.HTTP_401_UNAUTHORIZED)
+
+
+class AuthenticatedMovieAPITests(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.user = get_user_model().objects.create_user(
+            email="test@test.com", password="test1234"
+        )
+        self.client.force_authenticate(self.user)
```

Comment:
I see that you're using 'sample_movie' func without parameters in every test here, so you should move it to the setUp func where will be created 'self.movie' attribute